includes/export-action.php

```diff
@@ -347,6 +347,19 @@ function csv_export() {
                 
                 // Get all columns
                 let columns = window.SHAREDFUNCTIONS.get_json_cookie( 'fields_to_show_in_table', [] );
+
+                if ( columns.length === 0 ) {
+                    window.list_settings['post_type_settings']['fields']
```

Comment:
@prykon what is this line for?

Reply:
It was there for debugging purposes. Fix pushed.